lib/more_core_extensions/core_ext/hash/deletes.rb

```diff
@@ -15,6 +15,16 @@ def delete_nils
     def delete_blanks
       delete_if { |k, v| v.blank? }
     end
+
+    # Deletes all keys and subkeys that match +key+.
+    #
+    #   {:a => {:b => 2, :c => 3}}.deep_delete(:b) # => {:a => {:c => 3}}
+    #
+    def deep_delete(key)
```

Comment:
In https://github.com/ManageIQ/manageiq-gems-pending/issues/231 it says this should move to ManageIQ ApplicationController::Filter.  Did that change?  If so, can we update the issue?

Reply:
@bdunne True, but it seemed like it might be a useful method in general. Felt appropriate to put it here.

Reply:
Ok, I'll update the issue description.